examples/servicebus/servicebusnamespace.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: servicebus.azure.upbound.io/v1beta1
+kind: ServicebusNamespace
+metadata:
+  annotations:
+    meta.upbound.io/example-id: servicebus/v1beta1/servicebusnamespace
+  labels:
+    testing.upbound.io/example-name: example
+  name: example-y9w8 # Prevent naming conflict
```

@turkenh There is a naming conflict from Azure side when testing. I append a random strings in the end to avoid it. The name of servicebus namespace seems need to be global unique.

Please re-run the tests, thanks!

---

Uptest has a special template variable that you can use to generate random strings. Could you please try the following:
```suggestion
  name: example-${Rand.RFC1123Subdomain} # Prevent naming conflict
```

An example is here:
https://github.com/upbound/provider-azure/blob/4f0e6678c9027702a9cf3e21f67a8daa738e48be/examples/cosmosdb/table.yaml#L28